### PR TITLE
orientdb - bumping formula to 2.0 final release

### PR DESCRIPTION
**Pull request by poindextrose**
This simply bumps up the orientdb formula to the new 2.0 final release of OrientDB.

I also added a comment about where to find the automatically generated root password.

### CODE REVIEW

File: Library/Formula/orientdb.rb [hunk 1]
```diff
@@ -44,7 +30,10 @@ def install
     bin.install_symlink "#{libexec}/bin/gremlin.sh" => 'orientdb-gremlin'
   end
 
-  def caveats
-    "Use `orientdb <start | stop | status>`, `orientdb-console` and `orientdb-gremlin`."
+  def caveats; <<-EOS.undent
+    The root password is in /usr/local/opt/orientdb/libexec/config/orientdb-server-config.xml
```
Comment (jacknagel) [inline Library/Formula/orientdb.rb]: We can't hardcode `/usr/local`, use `#{opt_libexec}` instead.

File: Library/Formula/orientdb.rb [hunk 2]
```diff
@@ -44,7 +30,10 @@ def install
     bin.install_symlink "#{libexec}/bin/gremlin.sh" => 'orientdb-gremlin'
   end
 
-  def caveats
-    "Use `orientdb <start | stop | status>`, `orientdb-console` and `orientdb-gremlin`."
+  def caveats; <<-EOS.undent
+    The root password is in #{opt_libexec}/config/orientdb-server-config.xml after the orientdb is started for the first time.
```
Comment (mikemcquaid) [inline Library/Formula/orientdb.rb]: Please make sure this wraps nicely at 80 characters, thanks!

File: Library/Formula/orientdb.rb [hunk 3]
```diff
@@ -2,23 +2,9 @@
...
-    sha1 '84369ce34c749f2fab6bcbb90f92aaaa63deb44f'
-  end
-
-
-  bottle do
-    cellar :any
-    sha1 "ce4b7060fc28b4e686fae9cea7d6c016c988ef57" => :yosemite
-    sha1 "a6320a3d1b705aea9bdb5c73f7e15cfeae50b0eb" => :mavericks
-    sha1 "e58a8099e08788365b2d8ad4148e6e46ee1021f5" => :mountain_lion
-  end
```
Comment (mikemcquaid) [inline Library/Formula/orientdb.rb]: Leave the `bottle do` block as-is and we'll update it when we pull.

**Comment by jacknagel** [comment]
Looks like a different 2.0.1 PR was pulled.

**Pull request closed without merging**